app/importers/file_importers/downloader.rb

```diff
@@ -0,0 +1,32 @@
+class Downloader
```

Comment:
Is this a `CSVDownloader`?  It looks like it expects there be a `transformer` and that has a method `pre_cleanup_csv_size` etc.  Adding a comment about what the class is for could help.

Reply:
@kevinrobinson Good points, will implement.